src/uct/ib/base/ib_verbs.h

```diff
@@ -281,6 +281,7 @@ static inline int ibv_exp_cq_ignore_overrun(struct ibv_cq *cq)
 #    define IBV_TM_CAP_RC                   IBV_EXP_TM_CAP_RC
 #    define IBV_TM_CAP_DC                   IBV_EXP_TM_CAP_DC
 #  endif
+#  define UCT_RC_RNDV_HDR_LEN               sizeof(struct ibv_rvh)
```

Comment:
maybe move it to some  `rc_mlx5...` header? Or rename to `UCT_IB...`, since it is in `ib_verbs.h`